canonical-websites/www.ubuntu.com
# Redirect some common 404 in the last week

## Done

- Fixed some common 404 errors:
  - https://www.ubuntu.com/legal/dataprivacy/sso
  - https://www.ubuntu.com/legal/privacy-policy
  - https://www.ubuntu.com/legal/privacy-policy/third-parties
  - https://www.ubuntu.com/legal/dataprivacy/snap-store
  - https://www.ubuntu.com/releases
  - https://www.ubuntu.com/usn/rss.xml


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure the listed URLs are taking you to the expected content
